### PR TITLE
Removes Centcomm Junkfood Vending Machines

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Review
Removes the free food vending machines at centcomm.

Feedback Thread: https://forums.aurorastation.org/viewtopic.php?f=18&t=11821

### CODE REVIEW

**Comment by BurgerLUA** [comment]
Done.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -131,6 +131,7 @@
 	..()
 	new /obj/item/clothing/mask/breath( src )
 	new /obj/item/weapon/reagent_containers/inhaler/phoron_special(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/koisbar_clean(src)
```
Comment (Alberyk) [inline code/game/objects/items/weapons/storage/boxes.dm]: They already got their phoron inhaler, not sure why give vaurca more access to k'ois/phoron at roundstart, feels like this is going against stopping people from spawning with food suggestion.

Comment (skull132) [reply code/game/objects/items/weapons/storage/boxes.dm]: It basically is (hypocritical). If lunch boxes are power gaming by virtue of giving you access to an in-game resource you should normally work for, then this does exactly the same shit.

Comment (BurgerLUA) [reply code/game/objects/items/weapons/storage/boxes.dm]: I'll remove it then. I was on the fence about adding this in the first place.

**Comment by BurgerLUA** [comment]
Seems like a lot of people have issue with the lunchbox being removed, so I will remove that removal from this pr, and make another PR that refactors how lunchboxes work.

**Comment by skull132** [comment]
"How lunchboxes work" 🤔 

What did he mean by this.

**Comment by BurgerLUA** [comment]
@skull132 
Lunchboxes from loadouts will each have their own theme which contains a different assortment of junkfood. The types are: Vegan, Meat Eater, Balanced, and K'ois which can be specifically selected in the loadout menu. Self-explanatory.

**Comment by JackBewt** [comment]
This will not bring any more interaction to the chef. This will punish the entire station for there not being a chef, or for there not being a botanist to supply the chef quickly and reliably. Nothing in this PR will make anyone have to do anything more than grab food off the counter and keep walking. You are trying to force people to come to you to talk but this will not achieve even that goal.

**Comment by BurgerLUA** [comment]
There is no lunchbox customization, currently. It's completely random. I'm actually adding more customization so I don't know what on earth you're talking about.

**Comment by JackBewt** [comment]
It's not completely random. You can customise your lunchbox's contents. I always spawn with a jelly sandwich, candy apple, and bottle of water because I chose those to spawn in my lunchbox in character customization. 

You are wrong, and now I am not sure you have tried to use the lunchbox feature yourself?

**Comment by BurgerLUA** [comment]
Nope, someone told me it's customizable. I recall several months back people were complaining that their mom packed them k'ois waffles. I'll just limit it to junkfood.

**Comment by JackBewt** [comment]
why are you coding a nerf based on something someone told you a month ago when you don't even know how the current feature works.....

**Comment by ProbablyMenown** [comment]
Many of the Vaurca players use the vendor for kois bars. This might be a bit iffy, since nothing is being changed regarding kois for them.

**Comment by BurgerLUA** [comment]
The chef can provide Vaurca with filtered kois, and their phoron inhaler is pretty strong. If they run out of phoron ever, they can purchase a k'ois bar or seek assistance from chemistry.

**Comment by ProbablyMenown** [comment]
@BurgerLUA So your solution is for Vaurca to go to the vending machine to get more phoron?

**Pull request merged**